Content.Client/Instruments/InstrumentSystem.cs

```diff
@@ -179,12 +234,10 @@ public bool OpenMidi(EntityUid uid, ReadOnlySpan<byte> data, InstrumentComponent
         SetupRenderer(uid, false, instrument);
 
         if (instrument.Renderer == null || !instrument.Renderer.OpenMidi(data))
-        {
             return false;
-        }
 
+        SetMaster(uid, null);
```

Comment:
specify param name for null value

Reply:
I think that just makes code look bad, and if you have a decent IDE it shows the param name anyway:
![image](https://github.com/space-wizards/space-station-14/assets/6766154/b9b67614-453c-4a00-88ac-e73b4345e149)